### ISSUE TITLE
[Feature Request] Add a toggle for wall haptics

### ISSUE THREAD

**Issue by sanxi33** [BlqzingIce/HapticsTweaker#1]
Could you add a toggle for wall haptics?
This is really important to me. Thank you!

**Comment by BlqzingIce** [comment]
Tweaks55 already has a toggle for disabling wall clash entirely, getting rid of both the haptics and the particles. I could implement a setting to control the strength of the wall haptics but similar to saber clashes and burn marks, I don't think there's a need/want for configurability beyond a simple toggle, which is already implemented in Tweaks55 (and BS+ GameTweaker).

**Comment by BlqzingIce** [comment]
I got curious so I went and checked, turns out Tweaks55 and GameTweaker both handle the clashes a bit differently.  

Both can disable all particles for floor burn marks, saber clashes, and wall clashes (with the exception of floor burn mark sparkle particles with Tweaks55, probably a bug). However, while Tweaks55 also disables haptics for all three, GameTweaker only disables haptics for floor burn marks, just messing with particles only for saber and wall clashes. (EDIT: burn marks might not have any haptics to begin with...)  

This is definitely making me consider adding sliders for saber and wall clashes, for people who either just want to have weaker haptics, or for people who want to disable the haptics but keep the particle effects, which is not currently possible.
